<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
 {
    background-color: #F5D07F;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/Downloads/screen.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: #f5d17d;
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
	background-color: #FFFFFF
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
	background-color: #FFFFFF
}</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>230</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #2F54EB;  /* Синій фон */
    color: white;  /* Білий текст */
    border: 2px solid #1D39C4;  /* Темно-синя рамка */
    border-radius: 10px;  /* Закруглені кути */
    padding: 5px 15px;  /* Внутрішні відступи */
    text-align: center;
}

QPushButton:hover {
    background-color: #597EF7;  /* Світло-синій фон при наведенні */
}

QPushButton:pressed {
    background-color: #1D39C4;  /* Темно-синій фон при натисканні */
}

QPushButton:checked {
    background-color: #2F54EB;  /* Синій фон при позначенні */
    color: white;
    border: 2px solid #1D39C4;
}
</string>
        </property>
        <property name="text">
         <string>Autorization</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #ff0000;  
    color: white;  
    border-radius: 10px;  
    padding: 5px 15px;  
    text-align: center;
}

QPushButton:hover {
    background-color: #ff7772  
}

QPushButton:pressed {
    background-color: #ff0000;  
}

QPushButton:checked {
    background-color: #2F54EB;  /* Синій фон при позначенні */
    color: white;
    border: 2px solid #1D39C4;
}
</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
